src/mdbExtensionController.ts

```diff
@@ -742,6 +763,11 @@ export default class MDBExtensionController implements vscode.Disposable {
       EXTENSION_COMMANDS.MDB_CREATE_PLAYGROUND_FROM_TREE_VIEW,
       () => this._playgroundController.createPlayground()
     );
+    this.registerCommand(
+      EXTENSION_COMMANDS.MDB_CREATE_PLAYGROUND_FROM_CONNECTION_TREE_VIEW,
+      (treeItem: DatabaseTreeItem | CollectionTreeItem) =>
+        this._playgroundController.createPlaygroundFromTreeView(treeItem)
```

Comment:
[super nit] The naming here is a bit inconsistent/confusing. On line 348, we're asking "from tree item", whereas here it's "from connection tree view" - why do we have a connection here and why is it "view" and not "item"? Then on line 769, we're dropping "connection", but keep calling it "view", even though the argument is "item".